ext/libsharpyuv.cmd

```diff
@@ -12,7 +12,7 @@
 git clone --single-branch https://chromium.googlesource.com/webm/libwebp
 
 cd libwebp
-git checkout 048cfc
```

Comment:
I saw the error message "error: pathspec '048cfc' did not match any file(s) known to git" when I ran this script.